docs/user-guide/configure.md

```diff
@@ -413,6 +413,18 @@ For example:
 
 [More info](options.md#ignoredisables).
 
+### `configurationComment`
+
+You can set the `configurationComment` to change the prefix of stylelint commands used in `stylelint-disable` comments. This can be useful if you expect to have multiple instances of stylelint with different configurations running.
```

Comment:
```suggestion
You can set what configuration comments like `/* stylelint-disable */` start with. This can be useful if you use multiple instances of Stylelint with different configurations.

```